modules/serverless-jaeger-config.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * /serverless/serverless-tracing.adoc
+// */jaeger/jaeger_config/serverless-jaeger-integration.adoc
+
+[id="serverless-jaeger-config_{context}"]
+= Configuring Jaeger for use with {ServerlessProductName}
+
+.Prerequisites
+
+To configure Jaeger for use with {ServerlessProductName}, you will need:
+
+* Cluster administrator permissions on an {product-title} cluster.
+* A current installation of {ServerlessOperatorName} and Knative Serving.
+* A current installation of Jaeger.
+
+.Procedure
+
+. Create and apply the Jaeger custom resource:
++
+----
+$ cat <<EOF | oc apply -f -
+apiVersion: jaegertracing.io/v1
+kind: Jaeger
+metadata:
+  name: jaeger
+EOF
+----
+
+. Enable tracing on for Knative Serving, by editing the namespace and adding the tracing YAML:
++
+----
+$ oc edit ks -n knative-serving
+----
++
+.Tracing YAML example
+----
+tracing:
+  sample-rate: "0.1"
+  backend: zipkin
+  zipkin-endpoint: http://jaeger-collector.<JAEGER_NAMESPACE>.svc.cluster.local:9411/api/v2/spans
+  debug: "true"
+----
++
+.. The backend must be `zipkin`.
+.. The `zipkin-endpoint` must point to your `jaeger-collector` service endpoint.
++
+You can find this endpoint by getting the Jaeger URL:
++
+----
+$ oc get route
+----
++
+The service endpoint will be in the format `https://jaeger-<namespace>.apss.<snip>`.
```

Comment:
The `https://jaeger-<namespace>.apss.<snip>` is an URL for the Jaeger's web UI. The endpoint which we want to set to `zipkin-endpoint` is different from the URL for Jaeger's web UI.

Reply:
This one will need the `oc get route` command I believe? To easily fetch the URL. Example output would be great here.

Reply:
Umm.. Then, we should write the docs like "To access Jaeger's web UI, run `oc get route` and access  to https://jaeger-<namespace>.apss.yordomain.example.com".

Reply:
Right.